scratch/PCA/extract_data.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# In[61]:
+
+#!/usr/bin/env python
+
+"""
+DESCRIPTION
+
+    Script for scraping process parameters and defect parameters for PCA
+    Outfitted for use with 'process_parameters.csv'
+"""
+
+from __future__ import division
+from scipy import stats
+from matplotlib import pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import sys, os, glob
+import numpy as np
+import json
+
+def scrape_data(plate_list=['2'], feat_list=None, resp_var=None, stat_param=None):
+
+    # Retrieve all build features
+    all_build_data = np.genfromtxt('process_parameters.csv', delimiter=',',
+                                    dtype=None)
+    
+    # Obtain build features of interest
+    if feat_list==None:
+        feat_list = ['row', 'polar', 'col', 'azimuth', 'RD', 'TD']
```

Comment:
Consider using a `set` rather than a `tuple`.  It better expresses in the intention (unique elements) and makes it possible to manage defaults a little more easily:
```
>>> default_opts = {"row", "col"}
>>> user_opts = {"azimuth"}
>>> processing_opts = default_opts.copy()
>>> processing_opts.update(user_opts)
>>> processing_opts
set(['azimuth', 'col', 'row'])
```